### DISCUSSION TITLE
PMI and geometry

### DISCUSSION THREAD

**Discussion by wwll** [Open-Cascade-SAS/OCCT#92] [Q&A]
I would like to do the following with PMI: obtain the PMI, and access the STEP entities that contain the physical geometries that the PMI is referencing. How can I achieve achieve my purpose ?

**Comment by dpasukhi** [comment]
Hello. Result of the STEP importing stored in the XCAF structure.
It means, after reading you will lost the connection with STP entities.
But there is an option to connect them back if working session still alive. It require much more code then just working with XCAF.
Did you need access to the step model or the connection to the TopoDS_Shape will be enough?
If the connection with TopoDS_Shape  will enough -  i will share code samples. But for now could you explain your goal?

**Reply by wwll** [reply]
thanks for your reply!
Actually, I want to export a Step file from OCC ，and then try out the Step in NX, and I want to use NX to read the PMI and the geometry associated with the PMI. So I want to know if I create PMI in OCC,for example, a Radius  , and then export the PMI to a Step file. if the step file had been  imported into NX ,  could I  get the  geometry face associated with the PMI when I point to the PMI.

**Reply by wwll** [reply]
Hi ,dpasukhi。Does OCCT support Linking Annotations with Geometry (“Cross-Highlighting”) ?  When export a step,how to add draughting_model_item_association to Step?
